global/game_manager.gd

```diff
@@ -40,6 +40,12 @@ func AddItem(item_string):
 			items[i] = item_string
 			return
 
+func RemoveItem(item_string):
+	for i in range(0, 10):
+		if items[i] == item_string:
+			items[i] = null
+			return
+
```

Comment:
You can use the remove method on the array. It removes an element by it's value.

Reply:
Oh yeah that makes sense.